OpenRA.Mods.Common/Scripting/ScriptTriggers.cs

```diff
@@ -421,6 +422,48 @@ public void OnDiscovered(Actor self, Player discoverer, bool playNotification)
 			}
 		}
 
```

Comment:
You cannot explicitly implement interface members here.
These members are access via a `ScriptTriggers` instance, not an interface.